.travis.yml

```diff
@@ -58,31 +58,31 @@ script:
     case "${TARGET}" in
       amd64)
         docker run --rm \
-          --volume=`pwd`:/go/src/github.com/coreos/etcd gcr.io/etcd-development/etcd-test:go1.9.3 \
```

Comment:
Can we use the `TRAVIS_GO_VERSION` variable throughout `travis.yml` so we only have a single version number to update in the file?  I haven't tried it but it looks like it *MIGHT* work.

See https://docs.travis-ci.com/user/environment-variables/

Reply:
Great, let me try that. Thanks.